pkg/build/builder/util/dockerfile/dockerfile_test.go

```diff
@@ -306,6 +306,11 @@ COPY . /boot
 FROM centos:7`,
 			want: []string{"scratch", "centos:7"},
 		},
+		"FROM with ARG": {
+			in: `ARG TAG=7
+FROM centos:$TAG`,
```

Comment:
Could you fix this so the tabs match.

Reply:
It seems to me that they are the same as in previous block
```
^I^I"multiple FROM": {$
^I^I^Iin: `FROM scratch$
COPY . /boot$
FROM centos:7`,$
^I^I^Iwant: []string{"scratch", "centos:7"},$
^I^I},$
```
```
^I^I"FROM with ARG": {$
^I^I^Iin: `ARG TAG=7$
FROM centos:$TAG`,$
^I^I^Iwant: []string{"centos:7"},$
^I^I},$
```